study.md

```diff
@@ -87,8 +87,16 @@ Create the following movies with the given attributes.
 | 2 | The Core | 5.4 |
 
 ```ruby
-# your answer here
-```
+class Movie
```

Comment:
This is unnecessary.  We also never want to try to forcefully create records with `id`s since the database takes care of that for us, provided that primary keys are set.

```ruby
Movie.create(title: 'Battlefield Earth', rating: 2.4)
Movie.create(title: 'Sharknado', rating: 3.3)
Movie.create(title: 'The Core', rating: 5.4)
```